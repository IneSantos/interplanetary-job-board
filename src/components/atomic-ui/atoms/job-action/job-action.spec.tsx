import React from "react";
import { render } from '@testing-library/react';
import JobAction from "./job-action";

describe("JobAction atom tests", () => {
  test(`JobAction component render`, () => {
    const { container } = render(
      <JobAction />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
