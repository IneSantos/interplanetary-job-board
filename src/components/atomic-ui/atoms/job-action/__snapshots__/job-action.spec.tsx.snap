// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobAction atom tests JobAction component render 1`] = `
<a
  class="job-apply"
  href=""
>
  Apply
</a>
`;
